### PR TITLE
completed task get/api/topics

### PR DESCRIPTION
**Pull request by micatkinson**
add completed task, to update err handling block later

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    describe('GET', ()   =>   {
+        it('Status code 200', () =>  {
+            return request(app).get("/api/topics").expect(200);
+        });
+        it('should respond with an array of topic objects', ()   =>  {
+            return request(app)
+            .get("/api/topics")
+            .then(({body}) => {
+            const topics = body.topics;
+            expect(Array.isArray(topics)).toBe(true)
```
Comment (nc-alice-yang) [inline __tests__/app.test.js:27]: you need to look at the length other it will always autopass with a forEach

File: app.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const express = require("express")
+const Port = 3000
+
+const { getTopics } = require("./controllers/topics.controllers");
+
+const app = express();
+
+app.use(express.json());
```
Comment (nc-alice-yang) [inline app.js:8]: dont add code that you need yet as it is bad practice as a developer

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            topics.forEach((topic) => {
+                expect(typeof topic.slug).toBe('string');
+                expect(typeof topic.description).toBe('string')
+            });
+          });
+        });
+    }); 
+});
+
```
Comment (nc-alice-yang) [inline __tests__/app.test.js:36]: Please think of an error you may be able predict early on. what happens if you go to the wrong path?

**Review by alice-y-codes — COMMENTED** [review]
see comments

**Pull request merged**